src/coreclr/vm/unsafeaccessors.cpp

```diff
@@ -0,0 +1,805 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "common.h"
+
+#ifndef DACCESS_COMPILE
```

Comment:
Instead of end-to-end ifdef, you can place the file under `VM_SOURCES_WKS` in the cmakefile so that it is not compiled for DAC build at all.